x-pack/plugins/lens/public/xy_visualization/xy_config_panel/index.tsx

```diff
@@ -6,55 +6,50 @@
  */
 
 import './xy_config_panel.scss';
-import React, { useMemo, useState, memo, useCallback } from 'react';
+import React, { memo, useCallback } from 'react';
 import { i18n } from '@kbn/i18n';
 import { Position, ScaleType, VerticalAlignment, HorizontalAlignment } from '@elastic/charts';
-import { debounce } from 'lodash';
 import {
   EuiButtonGroup,
   EuiFlexGroup,
   EuiFlexItem,
   EuiFormRow,
   htmlIdGenerator,
-  EuiColorPicker,
-  EuiColorPickerProps,
-  EuiToolTip,
-  EuiIcon,
-  EuiPopover,
-  EuiSelectable,
-  EuiText,
-  EuiPopoverTitle,
 } from '@elastic/eui';
 import type { PaletteRegistry } from 'src/plugins/charts/public';
 import type {
   VisualizationLayerWidgetProps,
   VisualizationToolbarProps,
   VisualizationDimensionEditorProps,
   FramePublicAPI,
-} from '../types';
-import { State, visualizationTypes, XYState } from './types';
-import type { FormatFactory } from '../../common';
+} from '../../types';
+import { State, visualizationTypes, XYState } from '../types';
+import { FormatFactory } from '../../../common';
```

Comment:
nit: any reason to remove `import type` here? 🤔

Reply:
I think was just a bad merge.